packages/page-header/PageHeader.js

```diff
@@ -35,7 +35,7 @@ const PageHeader = ({
       payerId,
       className: 'float-md-right d-inline-block',
       skeletonProps: {
-        width: 180,
+        width: '180',
```

Comment:
Does changing it from number to string work? I would think you would need to add `180px` for it to still maintain the 180 pixel width.

All `react-loading-skeleton` is doing is assigning the width property to the component. [Here](https://github.com/dvtng/react-loading-skeleton/blob/master/src/skeleton.js#L53)

Reply:
Sorry confusing you guys here, the prop-type is incorrect. It should accept `string`, `number` or null. The current value there is correct.

Reply:
Good catch 😅 

https://github.com/Availity/availity-react/pull/194/commits/800c9375746a94c1b8ca2137eb85d124ffdcd98c